borg/archiver.py

```diff
@@ -2030,7 +2026,10 @@ def parse_args(self, args=None):
             args = self.preprocess_args(args)
         parser = self.build_parser(args)
         args = parser.parse_args(args or ['-h'])
-        update_patterns(args)
+        if args.func == self.do_create:
+            # need at least 1 path but args.paths may also be populated from patterns
```

Comment:
_recreate_ is not part of 1.0-maint, so it is not relevant here.
However for the recreate subcommand in master, the `paths` arguments are optional. So from the parsers point of view, nothing needs to be changed. However, the resulting args.paths will in future be a mixture of `R /pathname` arguments from pattern-files and `paths` commandline arguments, so the user will need to be careful that the same combination of root paths will be specified at recreate time.